platforms/esp32c3-buddy/.cargo/config.toml

```diff
@@ -1,21 +0,0 @@
-[target.riscv32imac-unknown-none-elf]
-runner = "espflash flash --monitor"
-
-[build]
-rustflags = [
-  "-C", "link-arg=-Tlinkall.x",
```

Comment:
@hawkw where did this end up living? I can't see it somewhere else, and the linker scripts need to be there.

Reply:
Nevermind found it